.gitignore

```diff
@@ -1 +1 @@
-output/**
+**output**
```

Comment:
Why is this required? It seems dangerously generic.

Reply:
My bad, I was trying to fix `tree --gitignore`, which kept showing output directory.
I fixed it now.